### PR TITLE
[zh] sync /security/authorization/authz-dry-run/

### PR DESCRIPTION
**Pull request by windsonsea**
Add zh text to this page:

```
content/zh/docs/tasks/security/authorization/authz-dry-run/
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/security/authorization/authz-dry-run/index.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
+---
+title: 预演
```
Comment (hanxiaop) [inline content/zh/docs/tasks/security/authorization/authz-dry-run/index.md:2]: Is `模拟运行` or `试运行` better？Not really sure about `预演`, and also I've never seen common Chinese translations of `Dry run`?

Comment (hanxiaop) [reply content/zh/docs/tasks/security/authorization/authz-dry-run/index.md:2]: May need some discussions of this one

Comment (rootsongjc) [reply content/zh/docs/tasks/security/authorization/authz-dry-run/index.md:2]: 我觉得翻译成”模拟运行“比较好，”预演“太难理解了。

Comment (kebe7jun) [reply content/zh/docs/tasks/security/authorization/authz-dry-run/index.md:2]: `模拟运行 ` +1.

Comment (windsonsea) [reply content/zh/docs/tasks/security/authorization/authz-dry-run/index.md]: Well noted. It's `模拟运行` now.

File: content/zh/docs/tasks/security/authorization/authz-dry-run/index.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
+---
+title: 模拟运行
+description: 展示如何在不强制执行的情况下模拟运行授权策略。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authorization/authz-dry-run/index.md:3]:
```suggestion
description: 展示如何在不实际执行的情况下，观察授权策略应用后的效果。
```

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**